terraform/aws/voiceOpsAWS.tf

```diff
@@ -105,6 +113,41 @@ resource "aws_route_table_association" "voiceOpsRta1c" {
 # Security Group Setup
 
 ## TBD
+resource "aws_security_group" "voiceOpsSSH" {
```

Comment:
I'd just have a generic security group for each set of servers (eg. voiceOpsMasters and voiceOpsNodes) and then apply the rules into them using aws_security_group_rule, because we need a few sets of ports opening for master<>node communications as well as having them wide open master<>master and node<>node, e.g.:

`resource "aws_security_group" "voiceOpsMastersSg" {
  name        = "masters.voiceops.capademy.com"
  vpc_id      = "${aws_vpc.k8sdemo-capademy-com.id}"
  description = "Security group for masters"

  tags = {
    KubernetesCluster = "voiceops.capademy.com"
    Name              = "masters.voiceops.capademy.com"
  }
}

resource "aws_security_group" "voiceOpsNodesSg" {
  name        = "nodes.voiceops.capademy.com"
  vpc_id      = "${aws_vpc.k8sdemo-capademy-com.id}"
  description = "Security group for nodes"

  tags = {
    KubernetesCluster = "voiceops.capademy.com"
    Name              = "nodes.voiceops.capademy.com"
  }
}

resource "aws_security_group_rule" "all-master-to-master" {
  type                     = "ingress"
  security_group_id        = "${aws_security_group.voiceOpsMastersSg.id}"
  source_security_group_id = "${aws_security_group.voiceOpsMastersSg.id}"
  from_port                = 0
  to_port                  = 0
  protocol                 = "-1"
}

resource "aws_security_group_rule" "all-master-to-node" {
  type                     = "ingress"
  security_group_id        = "${aws_security_group.voiceOpsNodesSg.id}"
  source_security_group_id = "${aws_security_group.voiceOpsMastersSg.id}"
  from_port                = 0
  to_port                  = 0
  protocol                 = "-1"
}

resource "aws_security_group_rule" "all-node-to-node" {
  type                     = "ingress"
  security_group_id        = "${aws_security_group.voiceOpsNodesSg.id}"
  source_security_group_id = "${aws_security_group.voiceOpsNodesSg.id}"
  from_port                = 0
  to_port                  = 0
  protocol                 = "-1"
}

resource "aws_security_group_rule" "https-external-to-master-0-0-0-0--0" {
  type              = "ingress"
  security_group_id = "${aws_security_group.voiceOpsMastersSg.id}"
  from_port         = 443
  to_port           = 443
  protocol          = "tcp"
  cidr_blocks       = ["0.0.0.0/0"]
}

resource "aws_security_group_rule" "master-egress" {
  type              = "egress"
  security_group_id = "${aws_security_group.voiceOpsMastersSg.id}"
  from_port         = 0
  to_port           = 0
  protocol          = "-1"
  cidr_blocks       = ["0.0.0.0/0"]
}

resource "aws_security_group_rule" "node-egress" {
  type              = "egress"
  security_group_id = "${aws_security_group.voiceOpsNodesSg.id}"
  from_port         = 0
  to_port           = 0
  protocol          = "-1"
  cidr_blocks       = ["0.0.0.0/0"]
}

resource "aws_security_group_rule" "node-to-master-tcp-4194" {
  type                     = "ingress"
  security_group_id        = "${aws_security_group.voiceOpsMastersSg.id}"
  source_security_group_id = "${aws_security_group.voiceOpsNodesSg.id}"
  from_port                = 4194
  to_port                  = 4194
  protocol                 = "tcp"
}

resource "aws_security_group_rule" "node-to-master-tcp-443" {
  type                     = "ingress"
  security_group_id        = "${aws_security_group.voiceOpsMastersSg.id}"
  source_security_group_id = "${aws_security_group.voiceOpsNodesSg.id}"
  from_port                = 443
  to_port                  = 443
  protocol                 = "tcp"
}`